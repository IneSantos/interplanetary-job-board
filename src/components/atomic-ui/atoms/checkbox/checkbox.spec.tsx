import React from "react";
import { render } from '@testing-library/react';
import Checkbox from "./Checkbox";

describe("Checkbox atom tests", () => {
  test(`Checkbox component render`, () => {
    const { container } = render(
      <Checkbox label="checkbox" parent="checkbox" clearFilter={false} onChange={jest.fn()}/>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
