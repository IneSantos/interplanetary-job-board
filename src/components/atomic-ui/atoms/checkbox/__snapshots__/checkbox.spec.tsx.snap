// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox atom tests Checkbox component render 1`] = `
<label>
  <span
    class="checkbox"
  >
    <input
      class="checkbox-input"
      data-parent="checkbox"
      name="checkbox"
      type="checkbox"
    />
    checkbox
  </span>
</label>
`;
